src/app/components/notification.ts

```diff
@@ -8,9 +8,13 @@ import {
 import {Store} from '@ngrx/store';
 import {NotificationActions} from '../actions/actions';
 import {Notification} from '../interfaces/notification.interface';
-// why does this freak out!!!
-import styles from './notificationStyles.styl';
+
 import 'rxjs/add/operator/do';
+// why does this freak out!!!
+// import styles from './notificationStyles.styl'; // you also have a typo
```

Comment:
there's a typo and when you use `import` then it's typescript aware and will be checked for type definitions